packages/driver/cypress/integration/commands/net_stubbing_spec.ts

```diff
@@ -1495,8 +1496,8 @@ describe('network stubbing', { retries: 2 }, function () {
           })
         }).then(() => {
           return $.get('/timeout').then((responseText) => {
+            expect(Date.now() - this.start).to.be.closeTo(expectedSeconds * 1000 + 475, 500)
             expect(responseText).to.eq(payload)
```

Comment:
this is what was causing the flake - rendering this giant 10kb `expect` causes some lag, which causes the `Date.now()` to sometimes be way off if CPU load is already high. widened the interval too